shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/ApplicationController.java

```diff
@@ -32,4 +39,63 @@
     description = "The endpoints to manage applications"
 )
 @SecurityRequirement(name = "bearer-key")
-public class ApplicationController extends BaseController<ApplicationService, Application> { }
+@Log4j2
+public class ApplicationController extends BaseController<ApplicationService, Application> {
+    @GetMapping("/name/{name}")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(security = { @SecurityRequirement(name = "bearer-key") })
+    public Application findOne(@PathVariable("name") String applicationName) {
```

Comment:
Can we also add the OpenAPI docs like e.g. like [here](https://github.com/terrestris/shogun/blob/main/shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/BaseController.java#L51-L77)?

Reply:
They should be added now :+1: